contracts/evoting/evoting.go

```diff
@@ -601,3 +616,64 @@ func (e evotingCommand) cancelElection(snap store.Snapshot, step execution.Step)
 
 	return nil
 }
+
+// verifyIdentity is a utility function to verify if a public key is associated
+// to a member of the roster or not. Returns no error if it's the case.
```

Comment:
I feel it would be better to have a function `isMemberOf(roster authority.Authority, pubKey crypto.Publikey) bool`

Reply:
=> Returns NO error if it's the case, ie. the publicKey is associated to a member of the roster. I agree that it should be "nil" instead of no error.

=> isMemberOf is a better name but if the return type is bool, when I find an error I cannot propagate it, would I just return false ?